lib/bracket-matcher-view.coffee

```diff
@@ -83,13 +95,13 @@ class BracketMatcherView
         matchPosition = @findMatchingStartPair(position, matchingPair, currentPair)
 
     if position? and matchPosition?
-      @startMarker = @createMarker([position, position.traverse([0, 1])])
-      @endMarker = @createMarker([matchPosition, matchPosition.traverse([0, 1])])
+      @startMarker = @createMarker([position, position.traverse([0, 1])], {brace: currentPair})
+      @endMarker = @createMarker([matchPosition, matchPosition.traverse([0, 1])], {brace: matchingPair})
       @pairHighlighted = true
     else
       if pair = @tagFinder.findMatchingTags()
-        @startMarker = @createMarker(pair.startRange)
-        @endMarker = @createMarker(pair.endRange)
+        @startMarker = @createMarker(pair.startRange, {brace: currentPair})
+        @endMarker = @createMarker(pair.endRange, {brace: mathcingPair })
```

Comment:
matching is spelled wrong here.  In addition, there's an extra space before the closing bracket.